packages/local_storage/lib/src/local_storage.dart

```diff
@@ -26,51 +21,55 @@ class LocalStorage {
       version: 1,
       onCreate: (db, version) {
         db.execute('''
-          CREATE TABLE $_tasksTableName(
-            $_tasksIdColumnName INTEGER PRIMARY KEY,
-            $_tasksContentColumnName TEXT NOT NULL,
-            $_tasksStatusColumnName INTEGER NOT NULL
+          CREATE TABLE tasks(
+            id INTEGER PRIMARY KEY,
+            conctent TEXT NOT NULL,
+            status INTEGER NOT NULL
```

Comment:
@kamilgadek 
make it a function (method) instead of getter and pass needed parameters, from where you would know the columns??